sql/src/main/java/org/apache/druid/sql/calcite/schema/InformationSchema.java

```diff
@@ -119,12 +122,14 @@
   private final Map<String, Table> tableMap;
   private final AuthorizerMapper authorizerMapper;
   private final String druidSchemaName;
+  private final String viewSchemaName;
 
   @Inject
   public InformationSchema(
       @Named(DruidCalciteSchemaModule.INCOMPLETE_SCHEMA) final SchemaPlus rootSchema,
       final AuthorizerMapper authorizerMapper,
-      @DruidSchemaName String druidSchemaName
+      @DruidSchemaName String druidSchemaName,
+      @ViewSchemaName String viewSchemaName
```

Comment:
I guess `NamedViewSchema` should probably use this injected name instead of its own name, and also `SqlResourceCollectorShuttle` should probably accept it as well since it does for druid schema name.

Reply:
It would take some additional changes in DruidPlanner and beyond to support passing in the view name, so I decided to just use a fixed name for the view schema, I don't think the additional complexity is worth it currently.